app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(32, 6)) == sum([5, 5, 5, 5, 6, 6])
```

Comment:
if you use sum() here just compare it with sum result no need to use sum again and put the list inside it